tracks/csharp/exercises/allergies/mentoring.md

```diff
@@ -79,9 +79,13 @@ public class Allergies
 
 - Instead of manually iterating over allergies, suggest to use LINQ.
 - The `List()` method should use the `IsAllergicTo()` method in its implementation, to prevent duplication of logic.
+  - Duplication is bad because it violates the [DRY Principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)  
 - The list of allergies can be created using `Enum.GetValues(typeof(Allergen))`, which does require a type cast to be able to work with it.
 - Although not strictly necessary, it is good practice to decorate the `Allergen` enumeration with the `[Flags]` attribute, to indicate that it can hold multiple values at once.
+  - [FlagsAttribute documentation](https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute) 
```

Comment:
Maybe use an inline link?

```suggestion
- Although not strictly necessary, it is good practice to decorate the `Allergen` enumeration with the [`[Flags]` attribute](https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute), to indicate that it can hold multiple values at once.
```